tests/gui/pages/main.spec.js

```diff
@@ -280,6 +280,14 @@ describe('Browser: MainPage', function () {
         mountpoints: [ drivePaths[1] ],
```

Comment:
Mildly unrelated to this particular PR, but mountpoints have the structure
```js
[{ path: '/private/var/vm', label: 'VM' }, ... ]
```
now

Reply:
Yeah I noticed but I left it so we can mend this in another PR, they're not essential to those tests atm either.